lib/autoyast.pm

```diff
@@ -59,6 +59,7 @@ sub expand_patterns {
             my @sle15;
             push @sle15, qw(base minimal_base enhanced_base apparmor sw_management yast2_basis);
             push @sle15, qw(x11 gnome_basic fonts)                                                       if check_var('DESKTOP', 'gnome');
+            push @sle15, qw(x11_yast)                                                                    if is_sle('15-sp2+');
```

Comment:
I think adding x11_yast if x11 isn't there (aka not DESKTOP=gnome above) is wrong. And IMO you can add that pattern also on ga and sp1. So just merge with the line above